src/aliceVision/depthMap/SemiGlobalMatchingRc.cpp

```diff
@@ -520,13 +519,13 @@ bool SemiGlobalMatchingRc::sgmrc(bool checkIfExists)
 
     if(_sp->exportIntermediateResults)
     {
-        DepthSimMap* depthSimMapFinal = _sp->getDepthSimMapFromBestIdVal(_width, _height, _volumeBestIdVal, _scale, _step, _rc, zborder, _depths);
+        DepthSimMap* depthSimMapFinal = _sp->getDepthSimMapFromBestIdVal(_width, _height, &_volumeBestIdVal, _scale, _step, _rc, zborder, _depths);
```

Comment:
just asking, any chances that it returns a `nullptr`? In case it would be better to deal with it. Also an `assert(depthSimMapFinal)` would not kill anyone... ;-)

Reply:
This function cannot return null.
The refactoring done [here](https://github.com/alicevision/AliceVision/pull/619) has been started for months now. It's quite close to be merged, but it will be done after the release.
So I would prefer to limit conflicts, as most of these things (like the bug fixed in this PR) are already fixed on the other side.